spec/models/user/stored_users_spec.rb

```diff
@@ -58,11 +58,13 @@
       stub_model.should_receive(:saved_uids).and_return saved_uids_stub
       stub_model.should_receive(:recent_uids).and_return recent_uids_stub
 
+      saved_uids_stub.should_receive(:stored_uid).at_least(:once).and_return(stored_uid)
       saved_uids_stub.should_receive(:order).with(:created_at).and_return saved_uids_stub
-      saved_uids_stub.should_receive(:reverse_order).and_return fake_stored_uid_entries[:saved]
+      saved_uids_stub.should_receive(:reverse_order).and_return [saved_uids_stub]
```

Comment:
We do try to update Rspecs to use the newer syntax when possible. I think these should be defined like:
```
# expect(object)to receive(:method_call).and_return(something)
expect(saved_uids_stub)to receive(:reverse_order).and_return([saved_uids_stub])
```

Reply:
Fixed the entire file using the new rspec syntax